### PR TITLE
depthai_demo UI and visualisation improvements. Ref #358

### PR DESCRIPTION
**Pull request by dhruvsheth-ai**
Changed branch to Main. Refer - https://github.com/luxonis/depthai/pull/358

### CODE REVIEW

File: depthai_demo.py [hunk 1]
```diff
@@ -170,13 +165,13 @@ def get_label_text(self, label):
...
             return self.labels[int(label)]
         else:
-            print(f"Label of ouf bounds (label_index: {label}, available_labels: {len(self.labels)}")
+            print(f"Label out of bounds (label_index: {label}, available_labels: {len(self.labels)}")
             return label
 
 
 class FPSHandler:
     def __init__(self, cap=None):
-        self.timestamp = time.monotonic()
```
Comment (VanDavv) [inline depthai_demo.py:179]: we should stick to `monotonic`

File: depthai_demo.py [hunk 2]
```diff
@@ -6,23 +6,18 @@
...
 import depthai as dai
-from depthai_helpers.version_check import check_depthai_version
-import platform
+import numpy as np
 
 from depthai_helpers.arg_manager import parse_args
 from depthai_helpers.config_manager import BlobManager, ConfigManager
 from depthai_helpers.utils import frame_norm, to_planar, to_tensor_result
 
-print('Using depthai module from: ', dai.__file__)
```
Comment (VanDavv) [inline depthai_demo.py:19]: This should stay

File: depthai_demo.py [hunk 3]
```diff
@@ -6,23 +6,18 @@
...
-print('Depthai version installed: ', dai.__version__)
-if platform.machine() not in ['armv6l', 'aarch64']:
-    check_depthai_version()
-
 conf = ConfigManager(parse_args())
 conf.linuxCheckApplyUsbRules()
+conf.adjustParamsToDevice()
```
Comment (VanDavv) [inline depthai_demo.py:20]: and this shouldn't be added here

**Review by VanDavv — CHANGES_REQUESTED** [review]
@dhruvsheth-ai I think something has gone wrong during the merge, as it seems that this PR would revert some of the changes we added to `main` recently.

**Comment by dhruvsheth-ai** [comment]
Is the updated file fine now @VanDavv ? The UI changes have been redone on the updated depthai_demo.py script.

**Review by VanDavv — APPROVED** [review]

**Comment by VanDavv** [comment]
Thanks @dhruvsheth-ai!

**Pull request merged**